routes/api/auth.js

```diff
@@ -3,6 +3,21 @@ const router = express.Router();
 const ctrl = require("../../controllers/user");
 const auth = require("../../validators/authenticate");
 const validateCredentials = require("../../validators/userdata");
+const multer = require("multer");
+const path = require("path");
+
+const tmpDir = path.join(__dirname, "..", "..", "tmp");
+
+const storage = multer.diskStorage({
+  destination: (req, file, cb) => {
```

Comment:
Jakby co to też może być `destination: tmpDir`
Callback jest bardziej po to, gdybyś coś więcej chciała w środku robić.